### ISSUE TITLE
'package:ffi/ffi.dart'; not found

### ISSUE THREAD

**Issue by ir2pid** [dart-lang/ffi#89]
Seems both android studio and vcs can't find the package after updating to 1.0.0

On line` import 'package:ffi/ffi.dart';` I get error:
`Target of URI doesn't exist: 'package:ffi/ffi.dart'.`

Where I use the Utf8 class I see error:
```
Error: Method not found: 'Utf8.fromUtf8'.                        
```

**Comment by dcharkes** [comment]
> Utf8.fromUtf8

This has been renamed to an extension method `toDartString()`, so that is expected. See [this diff](https://github.com/dart-lang/ffi/commit/afb6b2c3a15c50e8f12ac15f368c7f1560f2d2f1).

[`lib/ffi.dart` is there in v1.0](https://github.com/dart-lang/ffi/blob/f3346299c55669cc0db48afae85b8110088bf8da/lib/ffi.dart), are you sure this is not a cascading error from something else?

Could you upload a repro to GitHub?

**Comment by jerel** [comment]
I ran into something similar and saw this issue. In my case the message `'package:ffi/ffi.dart'; not found` in VSCode didn't seem to be accurate as it was fixed with `flutter clean/flutter pub get/etc` and an editor reload.

The fix for `.toNativeUtf8()` took a bit more experimentation though and turned out that since it's an extension method the variable must be strictly typed as `String`. For example:
```
listen((message) {
  String doc = message['doc'];
  native.execute(doc.toNativeUtf8());
});
```
instead of
```
listen((message) {
  native.execute(message['doc'].toNativeUtf8());
});
```
